lib/adyen/client.rb

```diff
@@ -137,21 +137,21 @@ def call_adyen_api(service, action, request_data, headers, version, with_applica
       request_data = request_data.to_json
 
       if action.is_a?(::Hash)
-        if action.fetch(:method) == "get"
+        if action.fetch(:method) == "get" || action.fetch(:method) == "GET"
```

Comment:
why would it sometimes be "get" one time, and "GET" on other times?

Reply:
This is because now we only generate checkout and for the old 'manual' services we have 'get'.
Will make a commit to change the template with a lowercase lambda in mustache to have everything the same.